lib/pitchfork/http_server.rb

```diff
@@ -1014,7 +1014,7 @@ def worker_loop(worker)
             else
               request_env = process_client(client, worker, prepare_timeout(worker))
               worker.increment_requests_count
-              @after_request_complete&.call(self, worker, request_env)
+              @after_request_complete&.call(self, worker, request_env || {})
```

Comment:
Should we just skip the callback instead?

I expect many callback would fail on an empty hash about as much as on `nil`, because some mandatory keys (e.g. `PATH_INFO` would be missing).

Technically, if we failed to parse the request, there's no "after request" stage.